test/onnx/test_onnx_opset.py

```diff
@@ -393,5 +392,31 @@ def forward(self, x):
 
         check_onnx_opsets_operator(MyModule(), x, ops, opset_versions=[9, 10])
 
+    def test_frobenius_norm(self):
```

Comment:
We are now keeping test_onnx_opset.py for testing special cases only. 
No need to add tests here in this PR.